incydr/cli/cmds/user_risk_profiles.py

```diff
@@ -0,0 +1,226 @@
+@click.argument("user", callback=user_lookup_callback)
+@click.pass_context
+def show(ctx: Context, user):
+    """
+    Show details for a user risk profile.
+
+    Accepts a user ID or a username.  Performs an additional lookup if a username is passed.
+    """
+    client = ctx.obj()
+    profile = client.user_risk_profiles.v1.get_user_risk_profile(user)
+    # TODO with output formatter, UserRiskProfile model
+    # tmp:
+    echo(profile.json())
+
+
+@user_risk_profiles.command(cls=IncydrCommand)
+@click.argument("user", callback=user_lookup_callback)
+@click.option(
+    "--start-date",
+    default=None,
+    help="Update a user's starting date. Accepts a date in yyyy-MM-dd (UTC) format.",
+)
+@click.option(
+    "--end-date",
+    default=None,
+    help="Update a user's departure date. Accepts a date in yyyy-MM-dd (UTC) format.",
+)
+@click.option(
+    "--notes", default=None, help="Update the optional notes on a user's profile."
+)
+@click.pass_context
+def update(ctx: Context, user, start_date=None, end_date=None, notes=None):
+    """
+    Update a user risk profile.
+
+    Accepts a user ID or a username.  Performs an additional lookup if a username is passed.
+    """
+    # TODO: pass 'CLEAR' to clear field?
+    if not any([start_date, end_date, notes]):
+        raise click.UsageError(
+            "At least one of --start-date, --end-date, or --notes is required to update a user risk profile."
+        )
+
+    client = ctx.obj()
+    try:
+        client.user_risk_profiles.v1.update(user, notes, start_date, end_date)
+    except DateParseError as err:
+        raise err
+
+
+# TODO: add_cloud_alias() or add_alias()
+@user_risk_profiles.command(cls=IncydrCommand)
+@click.argument("user", callback=user_lookup_callback)
+@click.argument("cloud-alias")
+@click.pass_context
+def add_cloud_alias(ctx: Context, user, cloud_alias):
+    """
+    Add a cloud alias to a user risk profile.
+
+    Accepts a user ID or a username.  Performs an additional lookup if a username is passed.
+
+    A cloud alias is the username an employee uses to access cloud services such as Google Drive or Box.
+    Adding a cloud alias allows Incydr to link a user's cloud activity with their Code42 username.
+    Each user has a default cloud alias of their Code42 username. You can add one additional alias.
+    """
+    client = ctx.obj()
+    client.user_risk_profiles.v1.add_cloud_aliases(user, cloud_alias)
+
+
+@user_risk_profiles.command(cls=IncydrCommand)
+@click.argument("user", callback=user_lookup_callback)
+@click.argument("cloud-alias")
+@click.pass_context
+def remove_cloud_alias(ctx: Context, user, cloud_alias):
+    """
+    Remove a cloud alias from a user risk profile.
+
+    Accepts a user ID or a username.  Performs an additional lookup if a username is passed.
+    """
+    client = ctx.obj()
+    client.user_risk_profiles.v1.delete_cloud_aliases(user, cloud_alias)
+
+
+# TODO, do we want 'bulk' to be a subcommand group
```

Comment:
In the original CLI we broke all `bulk` commands into a `bulk` subgroup.  Do we want to do that in this one, or is labeling the command itself with a `bulk` prefix better? It does have the benefit of displaying all bulk commands from the get-go if running something like `incydr user-risk-profiles --help`